run-qa-checks

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+echo ''
+echo 'Checking documentation build status'
+./runsphinx-build
+
+# check for broken link
+python -m http.server 8001 &> /dev/null & pid=$!
```

Comment:
I think you should make sure only the build directory of sphinx is served, otherwise the crawler will try to browse everything

Reply:
Okay thanks